### DISCUSSION TITLE
Tables font size decreasing in landscape

### DISCUSSION THREAD

**Discussion by CarlosHenriqueDamasceno** [mpdf/mpdf#1943] [Q&A] [answered]
I trying to show a table in PDF in landscape mode but the font size is decreasing as long as the table get bigger. I've already tried  use shrink_tables_to_fit = 0 and autosize.

**Comment by finwe** [comment, marked as answer]
Share code. See the [note in the documentation](https://mpdf.github.io/tables/tables.html):

> Note: mPDF will always resize tables if it is the only way to fit a row or whole table onto a full page.
